idaes/core/util/convergence/convergence.py

```diff
@@ -1,228 +0,0 @@
-##############################################################################
-# Institute for the Design of Advanced Energy Systems Process Systems
-# Engineering Framework (IDAES PSE Framework) Copyright (c) 2018-2020, by the
-# software owners: The Regents of the University of California, through
-# Lawrence Berkeley National Laboratory,  National Technology & Engineering
-# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia
-# University Research Corporation, et al. All rights reserved.
-#
-# Please see the files COPYRIGHT.txt and LICENSE.txt for full copyright and
-# license information, respectively. Both files are also available online
-# at the URL "https://github.com/IDAES/idaes-pse".
-##############################################################################
-"""
-This module is a command-line script for executing convergence evaluation
-testing on IDAES models.
-
-Convergence evaluation testing is used to verify reliable convergence of a
-model over a range of conditions for inputs and parameters. The developer of
-the test must create a ConvergenceEvaluation class prior to executing any
-convergence testing (see convergence_base.py for documentation).
-
-Convergence evaluation testing is a two step process. In the first step, a json
-file is created that contains a set of points sampled from the provided inputs.
-This step only needs to be done once - up front. The second step, which should
-be executed any time there is a major code change that could impact the model,
-takes that set of sampled points and solves the model at each of the points,
-collecting convergence statistics (success/failure, iterations, and solution
-time).
-
-To find help on convergence.py::
-
-   $ python convergence.py --help
-
-You will see that there are some subcommands. To find help on a particular
-subcommand::
-
-   $ python convergence.py  <subcommand> --help
-
-To create a sample file, you can use a command-line like the following (this
-should be done once by the  model developer for a few different sample sizes)::
-
-   $ python ../../../core/util/convergence/convergence.py create-sample-file
-         -s PressureChanger-10.json
-         -N 10 --seed=42
-         -e idaes.models.convergence.pressure_changer.
-             pressure_changer_conv_eval.PressureChangerConvergenceEvaluation
-
-More commonly, to run the convergence evaluation::
-
-   $ python ../../../core/util/convergence/convergence.py run-eval
-         -s PressureChanger-10.json
-
-Note that the convergence evaluation can also be run in parallel if you have
-installed MPI and mpi4py using a command line like the following::
-
-   $ mpirun -np 4 python ../../../core/util/convergence/convergence.py run-eval
-         -s PressureChanger-10.json
-
-"""
-import argparse
```

Comment:
John, where did this "documentation" end up?

Reply:
If we're talking sphinx docs, I don't know about it.  If we're talking about command line help, its in https://github.com/IDAES/idaes-pse/pull/231/files#diff-ba1405186222d7f01f8af3acb1b8f7f484254714bd53d2b5fb6b1b502548fa93 (idaes/commands/convergence.py), just search for help.